src/main/java/com/fasterxml/jackson/databind/AnnotationIntrospector.java

```diff
@@ -1397,6 +1397,39 @@ public JsonCreator.Mode findCreatorAnnotation(MapperConfig<?> config, Annotated
         return null;
     }
 
+    /**
+     * Method called to check if introspector is able to detect so-called Primary
+     * Creator: Creator to select for use when no explicit annotation is found
```

Comment:
```java
   /**
     * Method called to detect so-called <b>Primary Creator</b> (via {@link #findCreatorAnnotation}).
     * Primary Creator is the Creator to select for use when no explicit annotation is found.
     * .... // rest omitted
```

@cowtowncoder I was thinking maybe the first part of the JavaDoc could be this. But other than that, seems good.

Reply:
Hmmm. Reference to `findCreatorAnnotation()` seems misplaced since potential creators includes all constructors, most applicable static factory methods, not just ones annotated.
(in fact, the main point is to allow selecting non-annotated creators).